### ISSUE TITLE
Digi ID issues require immediate attention plz

### ISSUE THREAD

**Issue by DigiByte-Cam** [DigiByte-Core/digi-id-io#7]
1. The DigiByte mobile app on iOS is not available at all.
2. The DigiByte mobile app on Google Play is no longer supported by newer versions of Android.

Why keep the buttons to either App Store? Why not remove them? 

Say I’m recommending Digi-ID to you and when you go to the website you go to download the first app you see and you’re directed to a page that says no longer available. Can you see the issue with that?

There’s another page where it shows the Digi-ID/Antum-ID download link, however, the link is to Google play. When you press on the link it directs you to a page that says app is no longer available. That app is available on iOS App Store and that needs to be fixed.

**Comment by DigiByte-Cam** [comment]
Thanks to assistance from Jose Trejo, iv brought this task to bring to attention this current issue with Digi ID. Getting more involved …

**Comment by DGBNOOB** [comment]
The first recommendation is removing google play and app store buttons on the home page. By removing the buttons, users wont get the false impression that they can download the featured wallet. I think leaving it blank is better than having two buttons that redirect to pages no longer available. 

Second recommendation is to remove the DigiByte blockchain logo and two buttons on the get started page. An easy fix that doesn't require a lot of rearranging the whole card is replacing the current logo and buttons with the Digi-ID desktop logo and add a GitHub icon with link to Digi-ID desktop repo. The alternative is to remove the current logo and buttons and rearrange the icons/logos and links. 

Third recommendation is to replace google play button with iOS button and link to Digi-ID/AntumID app on App Store on the getting started page. There currently isn't an app on Google Play and it would be beneficial for the project to have them for iPhone and Android users, for now if we can have the correct button/link displayed that would be a good start.   

Home page
![google play app store buttons](https://github.com/DigiByte-Core/digi-id-io/assets/136190014/a341513b-d056-4051-87c6-a679263cf21f)

Getting started page
![getting started](https://github.com/DigiByte-Core/digi-id-io/assets/136190014/b191bfae-bab2-4976-96d0-cd1385ad6cda)

**Comment by DigiByte-Cam** [comment]
[DGB.pdf](https://github.com/DigiByte-Core/digi-id-io/files/15316203/DGB.pdf)

**Comment by DigiByte-Cam** [comment]
https://github.com/DigiByte-Core/digi-id-io/assets/169206889/50544fe8-4f30-4bf3-8a09-3edb772fd147


https://github.com/DigiByte-Core/digi-id-io/assets/169206889/19c022a8-23d4-498f-abfd-d466999049de

**Comment by LeroyGerrits** [comment]
It seems @Sbosvk already made some changes 2 months ago (see commit https://github.com/DigiByte-Core/digi-id-io/commit/9f8c3f1434c12d0a6f8e7990e5d743f1e4f1ff95) to hide the app store buttons in source; the changes to the website have just not been published yet.

**Comment by Sbosvk** [comment]
That's correct. There is something fishy with the deployment that I haven't been able to figure out. So those changes have been merged, they are just not live on the website.

**Comment by DGBNOOB** [comment]
> The first recommendation is removing google play and app store buttons on the home page. By removing the buttons, users wont get the false impression that they can download the featured wallet. I think leaving it blank is better than having two buttons that redirect to pages no longer available.
> 
> Second recommendation is to remove the DigiByte blockchain logo and two buttons on the get started page. An easy fix that doesn't require a lot of rearranging the whole card is replacing the current logo and buttons with the Digi-ID desktop logo and add a GitHub icon with link to Digi-ID desktop repo. The alternative is to remove the current logo and buttons and rearrange the icons/logos and links.
> 
> Third recommendation is to replace google play button with iOS button and link to Digi-ID/AntumID app on App Store on the getting started page. There currently isn't an app on Google Play and it would be beneficial for the project to have them for iPhone and Android users, for now if we can have the correct button/link displayed that would be a good start.
> 
> Home page ![google play app store buttons](https://private-user-images.githubusercontent.com/136190014/330654230-a341513b-d056-4051-87c6-a679263cf21f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NDk5MjMsIm5iZiI6MTcxNTk0OTYyMywicGF0aCI6Ii8xMzYxOTAwMTQvMzMwNjU0MjMwLWEzNDE1MTNiLWQwNTYtNDA1MS04N2M2LWE2NzkyNjNjZjIxZi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxN1QxMjQwMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYzcwZjA1YjNiY2FjYmM5NjQ0NjBmZjY4ODY2NGM0OTI5YjY2NzI2N2M2MGNlOTgxNTE3ODQ2NWYyY2JkMjUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.SkplO7zVCaADOcpSrtUzzK69BbAxXTdwgre7mWOGtmk)
> 
> Getting started page ![getting started](https://private-user-images.githubusercontent.com/136190014/330654228-b191bfae-bab2-4976-96d0-cd1385ad6cda.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NDk5MjMsIm5iZiI6MTcxNTk0OTYyMywicGF0aCI6Ii8xMzYxOTAwMTQvMzMwNjU0MjI4LWIxOTFiZmFlLWJhYjItNDk3Ni05NmQwLWNkMTM4NWFkNmNkYS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxN1QxMjQwMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MWZmNzU4NzAxYzE0ZWY2MTYxODMxOTY1YmI5MTczNmQ3Y2FmMTUxN2E4ZjZkZjhmM2I3ZjMyNjE4OGJiYjAyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7rAsUQd4Ns8cHV90t_ZKQ1rfMfEx0VUtWcEo1wnZI04)

@LeroyGerrits thank you for your input, I looked at the commits 9f8c3f1 and in this issue we are making three different recommendations to improve the website. I've been in communication with @Sbosvk and he explained the deployment issue. What are your thoughts on the recommendations?

**Comment by ycagel** [comment]
Any ideas on the deployment issue?

**Comment by DennisPitallano** [comment]
After checking the logs, it looks like there are some deprecated versions that need to be updated in the deployment file.

![Image](https://github.com/user-attachments/assets/161ab4e1-8ef4-4ea8-8394-4e58ae1a44a7)

**Comment by Sbosvk** [comment]
I think the real issue is some artifact versioning issue -- I was never able to figure out a way around it since any action resulted in ever further issues. That being said, at the time CD was new to me and I had never really deployed anything using any  versioning application. I'm sure there's an easy fix for whoever is brainy enough.

**Comment by ycagel** [comment]
Thanks to @DennisPitallano for fixing the deployment issue and implementing other changes.
